hpe3par_sdk/client.py

```diff
@@ -3680,3 +3725,106 @@ def removeVolumesFromVolumeSet(self, name, setmembers):
+
+    def remoteCopyGroupExists(self, name):
+        try:
+            self.getRemoteCopyGroup(name)
+        except exceptions.HTTPNotFound:
+            return False
+        return True
+
+    def remoteCopyGroupVolumeExists(self, remote_copy_group_name, volume_name):
+        try:
+            self.getRemoteCopyGroupVolume(remote_copy_group_name, volume_name)
+        except exceptions.HTTPNotFound:
+            return False
+        return True
+
+    def admitRemoteCopyLinks(self, targetName, source_port, target_port_wwn_or_ip):
+        """
+        Creating remote copy link from soure to target.
+        :param targetName - The name of target system
+        :type name: str
+        :source_port - Source ethernet/Fibre channel port
+        :type name: str
+        :target_port_wwn_or_ip- Target system's peer port WWN/IP
+        :type name: str
+        :returns: empty list if successful
+        """
+        return self.client.admitRemoteCopyLinks(targetName, source_port, target_port_wwn_or_ip)
+
+    def dismissRemoteCopyLinks(self, targetName, source_port, target_port_wwn_or_ip):
+        """
+        Dismissing remote copy link from soure to target.
+        :param targetName - The name of target system
+        :type name: str
+        :source_port - Source ethernet/Fibre channel port
+        :type name: str
+        :target_port_wwn_or_ip- Target system's peer port WWN/IP
+        :type name: str
+        :returns: empty list if successful
+        """
+        return self.client.dismissRemoteCopyLinks(targetName, source_port, target_port_wwn_or_ip)
+
+    def startrCopy(self):
+        """
+        Starting remote copy service
+        :returns: empty list if successful
+        """
+        return self.client.startrCopy()
+
+    def rcopyServiceExists(self):
+        """
+        checking rcopyservice status
+        :returns: True if status is 'Started'
+        :         False if status is 'Stopped'
+        """
+        return self.client.rcopyServiceExists()
+
+    def rcopyLinkExists(self,targetName,local_port,target_system_peer_port):
+        """
+        checking remote copy link from soure to target.
+        :param targetName - The name of target system
+        :type name: str
+        :source_port - Source ethernet/Fibre channel port
+        :type name: str
+        :target_port_wwn_or_ip- Target system's peer port WWN/IP
+        :type name: str
+        :returns: True if remote copy link exists
+        :         False if remote copy link doesn't exists
+        """
+        return self.client.rcopyLinkExists(targetName, local_port, target_system_peer_port)
+
+    def admitRemoteCopyTarget(self, targetName, mode, remote_copy_group_name, source_target_volume_pairs_list=[]):
+        """
+        Adding target to remote copy group.
+        :param targetName - The name of target system
+        :type name: str
+        :mode - synchronization mode
+        :type name: str
+        :remote_copy_group_name
```

Comment:
Change :remote_copy_group_name to :param remote_copy_group_name - Remote copy group name